test/runtests.jl

```diff
@@ -4,3 +4,4 @@ Random.seed!(123456)
 
 include("interface.jl")
 include("transform.jl")
+include("batch_norm.jl")
```

Comment:
Needs a new-line added. Possibly necessary to set your editor to ensure one is present?